english/shrooms/engage-man/toothless/enlightenment/eating-marshmellows.md

```diff
@@ -0,0 +1,21 @@
+You're feeling hungry anyway, so a little marshmellow can't hurt. You decide to eat through a good deal so you have plenty of energy.
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.